steps/test/unit/tReorderMSData.cc

```diff
@@ -21,6 +22,30 @@ using dp3::steps::Step;
 
 BOOST_AUTO_TEST_SUITE(reorder)
 
+bool compareMetaBinaries(const std::string& filename1,
```

Comment:
Can we consider more descriptive variable names?

e.g. maybe `filename_base` `filename_compare` `file_base` `file_compare` `begin_base` `begin_compare` instead of `filename` `file` `begin1`?
or perhaps `filename_expected` `filename_compare`, or anything along those lines, that way it is easier to tell at a glance which of the two contains the expected values and which of the two is being compared... This may also make output from failed boost unit tests (if using e.g. EQUAL_COLLECTIONS as suggested) more clear

Reply:
Done, added more meaningful variable names.